test/ReadLine.Tests/Abstractions/Console2.cs

```diff
@@ -1,50 +1,45 @@
-using Internal.ReadLine.Abstractions;
+using ReadLine.Abstractions;
 
 namespace ReadLine.Tests.Abstractions
 {
-    internal class Console2 : IConsole
+  internal class Console2 : IConsole
+  {
+    public Console2()
     {
-        public int CursorLeft => _cursorLeft;
-
-        public int CursorTop => _cursorTop;
-
-        public int BufferWidth => _bufferWidth;
-
-        public int BufferHeight => _bufferHeight;
-
-        private int _cursorLeft;
-        private int _cursorTop;
-        private int _bufferWidth;
-        private int _bufferHeight;
-
-        public Console2()
-        {
-            _cursorLeft = 0;
-            _cursorTop = 0;
-            _bufferWidth = 100;
-            _bufferHeight = 100;
-        }
-
-        public void SetBufferSize(int width, int height)
-        {
-            _bufferWidth = width;
-            _bufferHeight = height;
-        }
-
-        public void SetCursorPosition(int left, int top)
-        {
-            _cursorLeft = left;
-            _cursorTop = top;
-        }
-
-        public void Write(string value)
-        {
-            _cursorLeft += value.Length;
-        }
-
-        public void WriteLine(string value)
-        {
-            _cursorLeft += value.Length;
-        }
+      CursorLeft = 0;
+      CursorTop = 0;
+      BufferWidth = 100;
+      BufferHeight = 100;
```

Comment:
Let's leave the convention of putting `_` before private members and let's keep those private members and allow the public properties to return them